src/components/BookItem.jsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState, useRef } from 'react';
+import '../index.css';
+import { PropTypes } from 'prop-types';
+import ApexChart from './Chart';
+
+const BookItem = ({
+  bookProp, /* removeBook */ setUpdate, category,
+}) => {
```

Comment:
- [ ] As I mentioned above :point_up: please remove unwanted comments from your codebase because this makes your codebase messy if you want to check your previous code then check your GitHub commit history :+1: